package.json

```diff
@@ -143,13 +147,20 @@
     "@babel/runtime": "^7.10.5",
     "@types/jest": "^26.0.5",
     "@types/qs": "^6.9.2",
+    "@walletconnect/client": "^1.2.2",
+    "@walletconnect/qrcode-modal": "^1.2.2",
+    "@walletconnect/web3-provider": "^1.2.2",
     "babel-register": "^6.26.0",
+    "browserify": "^16.5.2",
+    "buffer": "^5.6.0",
     "marked": "^1.1.1",
     "pure-svg-code": "^1.0.6",
+    "radicle-contracts": "https://buildkite.com/organizations/monadic/pipelines/radicle-contracts/builds/72/jobs/e036d8ff-a6e5-4bf4-9d63-fd02bd297b2a/artifacts/b2ac2a23-6183-4219-ac35-1e36c444f7b2",
```

Comment:
Not for now, but this is a stable location for the period we are concerned with. We can’t point to the GitHub project directly (i.e. yarn add https://github.com/radicle-dev/radicle-contracts.git) because we need to create build artefacts (compile TS and the contracts) and yarn doesn't do this for git dependencies.

Reply:
We should definitely publish an npm package though soon.

Reply:
Ah! Does that mean that if we had a `.npmignore` file to the `Radicle Contracts` that pointing to the git project should work?

Reply:
AFAIU yes as the issue seems to be that yarn cleans up the build artefacts.

Reply:
will give it a shot

Reply:
gave it a try but it doesn't work. the issue being that [Yarn does not run `prepack` when installing Git dependencies](https://github.com/yarnpkg/yarn/issues/5047)